openpype/hosts/maya/api/lib.py

```diff
@@ -146,13 +146,15 @@ def suspended_refresh(suspend=True):
 
     cmds.ogs(pause=True) is a toggle so we cant pass False.
     """
-    original_state = cmds.ogs(query=True, pause=True)
+    original_state = None
```

Comment:
This would be better with just this at the beginning:

```python
if IS_HEADLESS:
    yield
    return
```
Then revert the rest of the function.

Reply:
Yeah, let's do that instead.